build2/KoreBuild.ps1

```diff
@@ -0,0 +1,100 @@
+Write-Host -ForegroundColor Green "Starting KoreBuild 2.0 ..."
+
+$RID = "win7-x64";
+if($env:PROCESSOR_ARCHITECTURE -eq "x86") {
+    $RID = "win7-x86";
+}
+
+$ToolsDirectory = $env:KOREBUILD_TOOLS_DIRECTORY
+if(!$ToolsDirectory) {
+    $ToolsDirectory = Join-Path $env:LOCALAPPDATA "Microsoft\AspNetCoreBuildTools\$RID"
```

Comment:
Agreed, this is a short-term solution. I'm not sure the best option here. I may take another look at having it installed in each repo and see what the impact of that is. The big problem is that we need to do a full restore of all the CoreCLR binaries **and** publish them to make sure CoreCLR works.